config/dictionaries/cloud.json

```diff
@@ -0,0 +1,56 @@
+[
+  {
+    "id": "aws",
+    "name": "AWS",
+    "shortName": "AWS",
+    "fullName": "AWS",
+    "platformName": "AWS IoT",
+    "platformUrl": "https://aws.amazon.com/iot",
+    "fullPlatformName": "AWS IoT",
+    "iotCoreName": "IoT Core",
+    "iotCoreUrl": "https://aws.amazon.com/iot-core",
+    "credentialType": "public key certificate",
+    "provisionRepoName": "aws-iot-provision",
+    "provisionRepoUrl": "https://github.com/balena-io-examples/aws-iot-provision",
+    "cloudFunctionName": "Lambda function",
+    "cloudFunctionFullNameStrong": "**AWS Lambda (cloud) function**",
+    "accountSetupLink": "https://github.com/balena-io-examples/aws-iot-provision#aws-setup",
+    "toolsSetupLink": "https://github.com/balena-io-examples/aws-iot-provision#tools-setup"
+  },
+  {
+    "id": "azure",
+    "name": "Azure",
+    "shortName": "Azure",
+    "fullName": "Microsoft Azure",
+    "platformName": "Azure IoT",
+    "platformUrl": "https://docs.microsoft.com/en-us/azure/iot-fundamentals/",
+    "fullPlatformName": "Microsoft Azure IoT",
+    "iotCoreName": "IoT Hub",
+    "iotCoreUrl": "https://docs.microsoft.com/en-us/azure/iot-hub/iot-concepts-and-iot-hub",
+    "credentialType": "public key certificate",
+    "provisionRepoName": "azure-iot-provision",
+    "provisionRepoUrl": "https://github.com/balena-io-examples/azure-iot-provision",
+    "cloudFunctionName": "Azure function",
+    "cloudFunctionFullNameStrong": "**Azure (cloud) function**",
+    "accountSetupLink": "https://github.com/balena-io-examples/azure-iot-provision#azure-iot-setup",
+    "toolsSetupLink": "https://github.com/balena-io-examples/azure-iot-provision#development-setup"
+  },
+  {
+    "id": "gcp",
+    "name": "Google Cloud",
+    "shortName": "GCP",
+    "fullName": "Google Cloud",
+    "platformName": "Cloud IoT",
+    "platformUrl": "https://cloud.google.com/solutions/iot",
+    "fullPlatformName": "Google Cloud (GCP) IoT",
+    "iotCoreName": "IoT Core",
+    "iotCoreUrl": "https://cloud.google.com/iot-core",
+    "credentialType": "public key pair",
+    "provisionRepoName": "gcp-iot-provision",
+    "provisionRepoUrl": "https://github.com/balena-io-examples/gcp-iot-provision",
```

Comment:
Same for these properties, I think we can optimize the properties further to reduce static data in this dictionary.

Reply:
Reduced dictionary size from 16 to 9 in 616c00c.